src/main/java/com/adobe/aem/compgenerator/javacodemodel/ImplementationBuilder.java

```diff
@@ -253,7 +255,9 @@ private void addGetter(JDefinedClass jc, JFieldVar jFieldVar) {
 
 
         if (jFieldVar.type().erasure().fullName().equals(List.class.getName())) {
-            getMethod.body()._return(codeModel.ref(Collections.class).staticInvoke("unmodifiableList").arg(jFieldVar));
+            JConditional ifNull = getMethod.body()._if(JExpr.ref(jFieldVar.name()).ne(JExpr._null()));
```

Comment:
Generated code ends up being `if (myvar!= null) {` with no space before the `!=`.  Any idea if that can be fixed?